source/classification1.md

```diff
@@ -22,6 +22,16 @@ from myst_nb import glue
 import numpy as np
 from sklearn.metrics.pairwise import euclidean_distances
 from IPython.display import HTML
+import altair as alt
+import os
+
+
+# Use PNG images in the PDF version of the books to make sure that they render
+if os.environ['BOOK_BUILD_TYPE'] == 'PDF':
```

Comment:
this will generate a key error when building in html -- you need to `export BOOK_BUILD_TYPE='HTML'` in the build html script